### ISSUE TITLE
NVDA is not announcing any update when user presses ENTER on "Show" and "Hide" control: A11y_Visual Studio Code Web Extensions_Github Pull request_Description_Usability

### ISSUE THREAD

**Issue by Rama8345** [microsoft/vscode-pull-request-github#6453]
### GIT Hub Tags:
#A11yTCS; #A11ySev4; #Visual Studio Code Web Extensions; #BM_Visual Studio Code Web Extensions_Web_Oct2024; #DesktopWeb; #A11yUsable; #NVDA; #Narrator; #Win11; #FTP; #Chrome; #Chromium Edge;

### Environment and OS details:
Application name: Visual Studio Code Web Extensions
Chrome Version: Version 130.0.6723.92 (Official Build) (64-bit)
URL: https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github
Screen reader: NVDA(2023.1), Narrator

### Reproduction Steps:

1. Start NVDA Open url https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github environment and login with git hub account.
2. TAB to 'Github Pull request' and press enter.
3. TAB to "All open" control and hit ENTER.
4. TAB to "Description" control and hit ENTER.
5. TAB to "Show" control and hit ENTER.
6. Observe the issue whether NVDA is announcing any update when user presses ENTER on "Show" and "Hide" control or not.

### Actual behavior:
NVDA is not announcing any update when user presses ENTER on "Show" and "Hide" control.

### Expected behavior:
Screen reader should announce update when user presses ENTER on "Show" and "Hide" control.

### User Impact:
Screen reader only users will not be able to recognize whether currently details are in hide mode or in show mode.

### Attachment:
![Image](https://github.com/user-attachments/assets/5879364a-734d-41cb-b0b7-e725496086da)
![Image](https://github.com/user-attachments/assets/843d3aa4-7e76-411f-b622-c0c1bf8c0213)
![Image](https://github.com/user-attachments/assets/2d4ac0ba-2f8c-4856-b384-1285c8d5d1e5)

**Comment by kupatkar99** [comment]
**Github tags:** A11yFixed;
